config/examples/nnf_v1alpha1_nnfdatamovementprofile.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfDataMovementProfile
+metadata:
+  name: default
```

Comment:
Call this one "template" so it doesn't collide with the site-customized default profile.

Reply:
Do I set `default:false` too then?

Reply:
yes

Reply:
The unpack-manifest tool will use this template to the create the default resource, and it will set default=true when it does that.